pkgs/servers/dns/nsd/default.nix

```diff
@@ -36,13 +35,16 @@ stdenv.mkDerivation rec {
     substituteInPlace nsd-control-setup.sh.in --replace openssl ${openssl}/bin/openssl
   '';
 
+  nativeBuildInputs = [
+    pkg-config
```

Comment:
This now adds `pkg-config` unconditionally, while before it was required `withSystemd` - is that intended?

Reply:
It probably makes no real world difference but we can add it back